lib/swarm/tracker/tracker.ex

```diff
@@ -719,14 +744,19 @@ defmodule Swarm.Tracker do
           :else ->
             # pid is dead, we're going to restart it
             case Strategy.key_to_node(state.strategy, name) do
+              :undefined ->
+                # No node available to restart process on, so remove registrartion
+                debug "no node available to restart #{inspect name}"
```

Comment:
My instinct is that this should be logged as a warning - it seems like a condition that we would definitely want logged when it occurs.